Cargo.toml

```diff
@@ -9,14 +9,14 @@ documentation = "https://dpc.github.io/rdedup-lib/"
 homepage = "https://github.com/dpc/rdedup"
 repository = "https://github.com/dpc/rdedup"
 readme = "README.md"
-
+exclude = ["lib/**/*"]
 
 [[bin]]
 name = "rdedup"
 path = "src/bin.rs"
 
 [dependencies]
-rdedup-lib = "0.0.1"
+rdedup-lib = { version = "0.0.1", path = "lib" }
```

Comment:
This works with both crates.io and local git? Nice. I was wondering if I can get it working like that.